src/tasks/env.mk

```diff
@@ -2,7 +2,7 @@
 # Environment variables come from https://github.com/Financial-Times/next-vault-sync
 .env:
 	@if [[ -z "$(shell command -v vault)" ]]; then echo "Error: You don't have Vault installed. Follow the guide at https://github.com/Financial-Times/vault/wiki/Getting-Started"; exit 1; fi
-	@if [[ -z "$(shell find ~/.vault-token -mmin -480)" ]]; then echo "Error: You are not logged into Vault. Try vault login --method github."; exit 1; fi
+	@if [[ -z "$(shell find ~/.vault-token -mmin -480)" ]]; then echo "Error: You are not logged into Vault. Try vault login --method github token=$VAULT_AUTH_GITHUB_TOKEN"; exit 1; fi
```

Comment:
I would put `token=\$VAULT_AUTH_GITHUB_TOKEN` so that the auth token isn't actually displayed, yet people can still copy/paste the output as a new command. Also means it won't get put into any history files.

Reply:
Thanks! Updated now!